src/state/slices/preferencesSlice/preferencesSlice.ts

```diff
@@ -105,5 +107,8 @@ export const preferences = createSlice({
     setWelcomeModal(state, { payload }: { payload: { show: boolean } }) {
       state.showWelcomeModal = payload.show
     },
+    setConsentBanner(state, { payload }: { payload: { show: boolean } }) {
```

Comment:
Also fine as is, but ditto more declarative to make this `setShowConsentBanner` and make the payload a boolean instead of a `{show: boolean}` object, since this is the only property of the payload